doc/modules/ROOT/pages/graph-object.adoc

```diff
@@ -84,6 +84,21 @@ include::ROOT:partial$/alpha-note.adoc[]
 
 Instead of projecting a graph from the Neo4j database it is also possible to construct new graphs using pandas `DataFrames` from the client.
 
+=== Syntax
+.Graph construct signature
+[opts="header",cols="1m,5m,6", role="no-break"]
+|===
+| Name                            | Type                                | Description
+| graph_name                      | str                                 | Name of the graph to be constructed.
+| nodes                           | Union[DataFrame, List[DataFrame]]   | One or more dataframes containing node data.
+| relationships                   | Union[DataFrame, List[DataFrame]]   | One or more dataframes containing relationship data.
+| concurrency                     | int                                 | Number of threads used to construct the graph.
```

Comment:
It would be nice to somehow denote that the default concurrency is 4 maybe?

Reply:
OK done